AzureADApplicationRegistration/functions/Set-AzureADResourceAccessV2.ps1

```diff
@@ -0,0 +1,91 @@
+
+function Set-AzureADResourceAccessV2 {
+    [CmdletBinding()]
+    param (
+        $resourceAccessDefinition = "D:\SourceCode\vh-book-hearing-client-provisioning\access.json" ,
+        $azureAdAppName = "vh_app_jb_preview"
```

Comment:
The default values will be removed once reviewd

Reply:
Should that be something more like <placeholder> instead of jb then? :P